Tiltfile

```diff
@@ -0,0 +1,81 @@
+load('ext://restart_process', 'docker_build_with_restart')
+
+optional_features = {
+    'feature-gate': ['feature-gate'],
+    'gateway': ['gateway'],
+    'api': ['api'],
+}
```

Comment:
TODO: backport changes in feature/logs-service to this PR, this tiltfile was better there

Reply:
done